drivers/hmis/spec/factories/hmis/hud/custom_client_addresses.rb

```diff
@@ -7,15 +7,18 @@
 FactoryBot.define do
   factory :hmis_hud_custom_client_address, class: 'Hmis::Hud::CustomClientAddress' do
     data_source { association :hmis_data_source }
-    user { association :hmis_hud_user, data_source: data_source }
     client { association :hmis_hud_client, data_source: data_source }
-    sequence(:AddressID, 100)
+    sequence(:AddressID) { |n| n + 100 }
 
     use { 'home' }
     address_type { 'postal' }
-    line1 { '123 Test St' }
+    sequence(:line1) { |n| "#{n + 123} Test St" }
     line2 { 'Apt 0' }
     city { 'Nowhere' }
-    state { 'XX' }
+    sequence(:state) { |n| ['KS', 'VT'][n % 2] }
```

Comment:
Thanks for all the factory improvements!

Reply:
Sure. I don't seem to be breaking other specs as far as I can tell.